docs/pt/docs/features.md

```diff
@@ -0,0 +1,202 @@
+# Recursos
+
+## Recursos do FastAPI
+
+**FastAPI** te oferece o seguinte:
+
+### Baseado em padrões abertos
+
+* <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI</strong></a> para criação de APIs, incluindo declarações de <abbr title="também conhecido como métodos HTTP, como POST, GET, PUT, DELETE">operações</abbr> de <abbr title="também conhecido como: endpoints, routes">caminho</abbr>, parâmetros, requisições de corpo, segurança etc.
+* Modelo de documentação automática com <a href="http://json-schema.org/" class="external-link" target="_blank"><strong>JSON Schema</strong></a> (já que o OpenAPI em si é baseado no JSON Schema).
+* Projetado em cima desses padrões, após um estudo meticuloso. Em vez de uma reflexão breve.
```

Comment:
Acredito não ser necessária a vírgula após a palavra `padrões`

Reply:
A tradução foi do trecho `Designed around these standards, after a meticulous study. Instead of an afterthought layer on top.`. Porém concordo com você e acho que essa pontuação soa melhor. Farei a mudança.